src/Core/RevEng.Core.60/ReverseEngineerRunner.cs

```diff
@@ -305,6 +290,32 @@ public static void RetryFileWrite(string path, string finalText)
         }
 #endif
 
+        private static void ValidateOptions(ReverseEngineerCommandOptions options, List<string> warnings)
+        {
+            if (options.UseDatabaseNames && options.CustomReplacers?.Count > 0)
+            {
+                warnings.Add($"'use-database-names' / 'UseDatabaseNames' has been set to true, but a '{Constants.RenamingFileName}' file was also found. This prevents '{Constants.RenamingFileName}' from functioning.");
+            }
+
+            if (options.UseT4 && options.UseT4Split)
+            {
+                warnings.Add("Both UseT4 and UseT4Split are set to true.  Only one of these should be used, UseT4Split will be used.");
+                options.UseT4 = false;
+            }
+
+            if (options.UseDbContextSplitting)
+            {
+                warnings.Add("UseDbContextSplitting (preview) is obsolete, please switch to the T4 split DbContext template.");
+                options.UseDbContextSplitting = false;
+            }
+
+            if (options.UseT4Split && options.UseDbContextSplitting)
```

Comment:
Isn't this logically impossible to happen due to the previous conditional?  With 306, `UseDbContextSplitting` can never be anything but false at this point.

Reply:
Fixed in next PR